apps/backend/sample-files/transaction.json

```diff
@@ -1,176 +1,173 @@
 [
-    {
-      "transaction_id": "72cbe1da-b08e-49d4-9a87-69afd660d14b",
-      "transaction_date": "2023-01-01",
-      "transaction_time": "13.33.31.875973",
-      "fiscal_close_date": "2023-01-02",
-      "amount": 25.20,
-      "void_indicator": false,
-      "miscellaneous": {
-        "employee_id": "SC61350"
+  {
+    "transaction_id": "72cbe1da-b08e-49d4-9a87-69afd660d14b",
+    "transaction_date": "2023-01-01",
+    "transaction_time": "13.33.31.875973",
+    "fiscal_close_date": "2023-01-02",
+    "total_payment_amount": 25.2,
```

Comment:
Naming change. I can revert, however, I think this helps to differentiate between the payment "amount"